linter/yamllint_rules.yml

```diff
@@ -1,10 +1,26 @@
 extends: default
 rules:
-  document-start: disable
+  document-start:
+    level: error
+    present: false
```

Comment:
Some files contain starting `----` but most of them go just straight to YML content. Although it is not an error I think it is good to enforce it because of uniformity, wdyt?

Reply:
👍 here there's nothing to doc, less clutter